src/import/sst_service.rs

```diff
@@ -203,6 +204,20 @@ impl<Router: RaftStoreRouter> ImportSst for ImportSSTService<Router> {
         let timer = Instant::now_coarse();
 
         // Make ingest command.
+        let handle = get_cf_handle(&self.engine, CF_DEFAULT).unwrap();
+        if let Some(n) = get_cf_num_files_at_level(&self.engine, handle, 0) {
+            let options = self.engine.get_options_cf(handle);
+            let slowdown_trigger = options.get_level_zero_slowdown_writes_trigger();
+            // Leave enough buffer to tolerate heavy write workload,
+            // which may flush some memtables in a short time.
+            if n > u64::from(slowdown_trigger) / 2 {
```

Comment:
seem the logic is the same as that in `region.rs`, can we extract a common function?